tools/find-inactive-collaborators.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+// Identify inactive collaborators. "Inactive" is not quite right, as the things
+// this checks for are not the entirety of collaborator activities. Still, it is
+// a pretty good proxy. Feel free to suggest or implement further metrics.
+
+const SINCE = process.argv[2] || '6 months ago';
+
+const childProcess = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+// Retrieve all commit authors during the time period.
+const authors = childProcess.execSync(
+  `git shortlog -n -s --since="${SINCE}"`,
+  { cwd: path.resolve(__dirname, '..'),
+    encoding: 'utf8',
+    stdio: ['inherit', 'pipe', 'pipe'] }
+).split('\n')
+ .map((line) => line.trim(' ').split('\t')[1])
```

Comment:
```suggestion
 .map((line) => line.trim().split('\t', 3)[1])
```

Reply:
I used `2` instead of `3`. Let me know if I'm missing something!